src/components/organization/organization.service.ts

```diff
@@ -31,9 +31,9 @@ export class OrganizationService {
     try {
       await this.db.createNode({
         session,
+        type: Organization.classType,
         input: { id, ...input },
         acls,
-        baseNodeLabel: 'Organization',
         aclEditProp: 'canCreateOrg',
```

Comment:
Should we maybe rename this edit prop across the board to `canCreateOrganization` for consistency's sake?

Reply:
Probably. It was referenced in a few different places. I didn't want to change all of them here.